Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -68,15 +69,15 @@ def _search(
         params[name] = value
         result = _validate(data, type(learner)(**params), scorer)
         scores.append((value, result))
-    return scores, scorer.name, fitted_parameter_props.tick_label
+    return scores, scorer.name, fitted_parameter_props.label
```

Comment:
I think it's nicer to write a full label as the axis label, and put just number in the ticks.